lib/vagrant-libvirt/templates/domain.xml.erb

```diff
@@ -239,6 +239,13 @@
 	  .join(' ')
 	-%>/>
       </source>
+    <%- if [pci[:guest_domain], pci[:guest_bus], pci[:guest_slot], pci[:guest_function]].any? -%>
```

Comment:
I think it should be possible to adjust this block a bit to only need it to be aware of keys starting with the `guest_` suffix and allow control of what is acceptable via the config definition. At least until something more explicit is needed it's a little easier to only need to modify one place to add new attributes rather than keeping multiple in sync.

Reply:
I see what you mean, that would certainly be neater. My ruby knowledge still doesn't extend far beyond cut and paste examples so any suggestions are more than welcome :)